src/graphql/queries/grant.ts

```diff
@@ -67,6 +69,14 @@ export const QUERY_GRANT = gql`
         }
       }
       applicants {
+        contributors {
+          amount
```

Comment:
1. You don't need the amount and times contributed of the individual contributors
2. It's missing the pagination input on the contributors `field`
3. It's missing the `contributorsCount` field